src/controllers/LearningProvider/learningProviderController.ts

```diff
@@ -64,9 +70,19 @@ export class LearningProviderController {
 				})
 			}
 
-			await self.learningCatalogue.createLearningProvider(learningProvider)
+			const newLearningProvider = await self.learningCatalogue.createLearningProvider(learningProvider)
+
+			response.redirect('/content-management/learning-providers/' + newLearningProvider.id)
+		}
+	}
 
-			response.redirect('/content-management/learning-providers')
+	private async getLearningProviderAndRenderTemplate(request: Request, response: Response, view: string) {
+		const learningProviderId: string = request.params.learningProviderId
+		const learningProvider = await this.learningCatalogue.getLearningProvider(learningProviderId)
+		if (learningProvider) {
+			response.render(view, {learningProvider})
+		} else {
+			response.sendStatus(404)
```

Comment:
Duplicated from CancellationPolicyController. Should leverage `app.param` from express and remove duplication.